src/Sulu/Bundle/WebsiteBundle/Resources/views/Analytics/matomo/head-close.html.twig

```diff
@@ -1,12 +1,12 @@
 <script>
-    var _paq = _paq || [];
+    var _paq = window._paq = window._paq || [];
     _paq.push(['trackPageView']);
     _paq.push(['enableLinkTracking']);
     (function() {
         var u="{{ analytics.content.url|trim('/') }}";
-        _paq.push(['setTrackerUrl', u+'/piwik.php']);
+        _paq.push(['setTrackerUrl', u+'matomo.php']);
```

Comment:
The forward slash has been removed here.

Reply:
You are right. I copied the latest code from the Matomo docs without checking what the u variable will contain. Thanks!